tests/sv-comp/dune.inc

```diff
@@ -0,0 +1,902 @@
+
```

Comment:
As far as I understand, this is auto-generated? Can you maybe in the generator print a comment into here to tell people not to manually do things here?
While people are usually familiar with `autoconf` and friends, this generation mechanism probably is new for most.